src/http/request.cr

```diff
@@ -98,6 +98,16 @@ class HTTP::Request
     value
   end
 
+  # Return request host from headers
+  def host
+    @headers["Host"]?.try &.to_s.sub(/:\d+\z/, "")
```

Comment:
I was going to suggest the same :-)

Simply find the index of `':'` and if such index exist make a substring until that index.